manual/manual_compiler.rst

```diff
@@ -587,9 +626,18 @@ After solving for the device connectivity, we then need to restrict what optimis
     [tk1(0, 0, 1.5) node[0];, tk1(0, 0, 1.5) node[1];, tk1(0, 0, 1.5) node[2];, tk1(0, 0, 1.5) node[3];, CX node[1], node[0];, tk1(0, 0, 0.5) node[0];, CX node[1], node[2];, CX node[1], node[3];, tk1(0, 0, 0.5) node[2];, tk1(0, 0, 0.5) node[3];, CX node[3], node[4];, CX node[1], node[3];, CX node[3], node[4];, CX node[4], node[3];, CX node[3], node[4];, CX node[3], node[1];]
     9
 
+
+.. Note::
+    :py:class:`FullPeepholeOptimise` takes an optional allow_swaps argument. This is a Boolean flag to indicate whether :py:class:`FullPeepholeOptimise` should preserve the circuit connectivity or not. If set to False the pass will presrve circuit connectivity but the circuit will general be less optimal than if connectivity was ignored.
+    
+    :py:class:`FullPeepholeOptimise` also takes an optional target_2qb_gate argument to specify whether to target the {:py:class:`OpType.TK1`, :py:class:`OpType.CX`} or {:py:class:`OpType.TK1`, :py:class:`OpType.TK2`} gateset.
+
+.. Note::
+    Prevous iterations of :py:class:`FullPeepholeOptimise` did not apply the :py:class:`ThreeQubitSquash` pass. There is a :py:class:`PeepholeOptimise2Q` pass which applies the old pass sequence with the :py:class:`ThreeQubitSquash` pass excluded.
+
 .. `Backend.default_compilation_pass` gives a recommended compiler pass to solve the backend's constraints with little or light optimisation
 
-Also in this category, we have the :py:meth:`Backend.default_compilation_pass()` which is run by :py:meth:`Backend.get_compiled_circuit`. These give a recommended compiler pass to solve the :py:class:`Backend` 's constraints with a choice of optimisation levels.
+Also in this category of pre-defined sequences, we have the :py:meth:`Backend.default_compilation_pass()` which is run by :py:meth:`Backend.get_compiled_circuit`. These give a recommended compiler pass to solve the :py:class:`Backend` 's constraints with a choice of optimisation levels.
```

Comment:
```suggestion
Also in this category of pre-defined sequences, we have the :py:meth:`Backend.default_compilation_pass()` which is run by :py:meth:`Backend.get_compiled_circuit`. These give a recommended compiler pass to solve the :py:class:`Backend` 's constraints with a choice of optimisation levels.
```
(It might not be obvious what this change is: the space before "'s" was changed to unicode 0x202f NARROW NO-BREAK SPACE.)

Reply:
I think this should be fixed now. Thanks.

Done.

Reply:
My bad, I think this may have been correct before -- looks like it was 0x202f and is now gone?

Reply:
Okay I've changed it back to how it was before.

Reply:
Hmm, seems to be a normal space again now?

Easy way to fix would be to copy and paste ":py:class:`Predicate` s" from line 23 and edit.

Reply:
Oh oops you're right. Should be fixed now hopefully.